bin/start-coordinator.sh

```diff
@@ -60,15 +60,16 @@ set -u
 
 echo "class path is $CLASSPATH"
 
-JVM_ARGS=" -server \
-          -Xmx${XMX_SIZE:-8g} \
-          -Xms${XMX_SIZE:-8g} \
+UNIFFLE_COORDINATOR_XMX_SIZE=${UNIFFLE_COORDINATOR_XMX_SIZE:-${XMX_SIZE:-"8g"}}
+UNIFFLE_COORDINATOR_DEFAULT_JVM_ARGS=${UNIFFLE_COORDINATOR_DEFAULT_JVM_ARGS:-" -server \
```

Comment:
The default gc log jvm Args is defined by someone but not suitable for all use cases, it is verbose and I want to remove or add some variables for different case. So we can give a default gc log args, but we'd better not forbidden users to do customize configuration, otherwise, there could be many customized scripts.